etl/src/main/scala/etl/ImageExtraction.scala

```diff
@@ -0,0 +1,42 @@
+package etl
+
+import com.gu.contentapi.client.model.v1.{ Asset, Content, Element }
+import com.gu.recipeasy.models.{ Image }
+
+object ImageExtraction {
+
+  def getImages(content: Content, articleId: String): Seq[Image] = {
+    val images = findImageElements(content)
+    images match {
+      case Some(images) => images.flatMap(i => extractImageData(i, articleId))
+      case None => Nil
+    }
+  }
+
+  private def findImageElements(content: Content): Option[Seq[Element]] = {
```

Comment:
This can return a Seq[Element] as opposed to an Option[Seq[Element]] and return Nil if it doesn't find any imageElements given you are matching on None above and returning Nil.